### PR TITLE
Add multiline text attribute values support

### PR DESCRIPTION
**Pull request by sergiosmcr**
Add the capability to specify a multiline text as value of an attribute by using ```.

For example:
~~~
[
  message=```
    Con la luz del sol
    Se derriten mis alas
    Solo encuentro en la oscuridad
      Lo que me une
      Con la ciudad de la furia
  ```
] dut1
~~~

The above text will be parsed as:
```python

    'environment': OrderedDict(),
    'links': [],
    'nodes': [
        {
            'attributes': OrderedDict([
                ('message', '\nCon la luz del sol\nSe derriten mis alas\nSolo encuentro en la oscuridad\n  Lo que me une\n  Con la ciudad de la furia\n'),
            ]),
            'nodes': ['dut1'],
        },
    ],
    'ports': [],
}
```

### CODE REVIEW

File: lib/pyszn/injection.py [hunk 1]
```diff
@@ -132,7 +132,8 @@ def parse_attribute_injection(
         injection_spec = loads(fd.read())
 
     # Cache the parsed topologies to not parse them again and again
-    # Key is a topology string and return is a dictionary of the parsed topology
+    # Key is a topology string and return is a dictionary of the parsed
```
Comment (dajose) [inline lib/pyszn/injection.py:135]: why this change?

File: lib/pyszn/injection.py [hunk 2]
```diff
@@ -179,7 +182,9 @@ def parse_attribute_injection(
 
                 # Nodes
                 if 'nodes' in modifier:
-                    for node in _expand_nodes(parsed_topology, modifier['nodes']):
+                    for node in _expand_nodes(
```
Comment (dajose) [inline lib/pyszn/injection.py:185]: why this change?

**Review by dajose — APPROVED** [review]

File: lib/pyszn/injection.py (hunk 1 above)
Comment (sergiosmcr) [reply lib/pyszn/injection.py:135]: flake8 complains

File: lib/pyszn/injection.py (hunk 2 above)
Comment (sergiosmcr) [reply lib/pyszn/injection.py:185]: flake8.
I ran tests and flake8 complained, so I fixed this code. I am guessing CIT does not run the tests
